OJP/OJP_Trips.xsd

```diff
@@ -723,6 +729,47 @@
 			<xs:enumeration value="downAndUp"/>
 		</xs:restriction>
 	</xs:simpleType>
+	<xs:simpleType name="AccessibilityFeaturesTypeEnumeration">
+		<xs:annotation>
+			<xs:documentation>Allowed values for AccessibilityFeatures.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:string">
+			<!--For mobility impairments...-->
+			<xs:enumeration value="lowFloorAccess"/>
```

Comment:
Are those comments OK or against the accepted coding conventions? Other solutions?

Reply:
OK